### ISSUE TITLE
Mixer Page - Add multiple Expo settings

### ISSUE THREAD

**Issue by leckert123** [FrSkyRC/ETHOS-Feedback-Community#87]
It is common to set up double or triple rates, each with their own % rate/weight, as well as with a % Expo.  As an example, I want set up triple rates and expo on switch D. Setting 3 rates is simple. It's all on one page in the Aileron mixer setting. However, there is only a single expo % on available that page.  We just need to add multiple Expo settings and, like rate/weight, be able to assign a switch position to each.

Here's my objective, which can be accomplished easily in FrOS, JETI, Futaba, etc on a single page: 

**Full 3D Rates, lots of expo:**
Switch D up - Rate =100%,, Expo = 65%
![screenshot-2021-01-04-33604](https://user-images.githubusercontent.com/41870928/114958342-c8d5b680-9e30-11eb-8b66-22cb7772fe2e.jpg)
![screenshot-2021-01-04-33615](https://user-images.githubusercontent.com/41870928/114958344-cb381080-9e30-11eb-81b1-0a576fc0bcb8.jpg)
![screenshot-2021-01-04-33629](https://user-images.githubusercontent.com/41870928/114958351-ce330100-9e30-11eb-97a1-d12781d81a83.jpg)

**Mid Rate, mid less expo**
Switch D middle - Rate = 60%, Expo = 40%

**Low Rate, least expo**
Switch D down, Rate = 40%, Expo = 25%

For most menu-driven radios (including FrOS) this is all very easily handled on the same menu page. Having to add additional ailerons is not going to be easy or intuitive for non OpenTX users who FrSky seeks to attract.  Coming from FrOS, this seems like a huge step backwards.

https://www.rcgroups.com/forums/showpost.php?p=46943163&postcount=5411
https://www.rcgroups.com/forums/showpost.php?p=46944061&postcount=5432
https://www.rcgroups.com/forums/showpost.php?p=46944191&postcount=5436

**Comment by EngelMT** [comment]
This is an very important feature and must be added in next update!

**Comment by Hartinger** [comment]
It would even be better if the selection could be made either via a switch, or via flight modes.
E.g. NORMAL/AKRO/3D flight mode for an aerobatic plane.

**Comment by EngelMT** [comment]
@Hartinger 

That it is possible via fligth modes. But in the moment we missing the possiblilty to set a seperate curve for each flight mode. And we are missing to set differant curves depend on the weight. Weight works in mixers like a Dual Rate. You can set different weights but with only one simelar curve.

**Comment by Hartinger** [comment]
You are quite right.
In 1.0.1 it was not yet possible to make the weighting flight mode dependant.
I missed that this has been introduced in 1.0.2, so my comment is kind of irrelevant.

**Comment by DaySpeed955i** [comment]
You can however use a channel as input which can have different values for expo with either a switch or a pot.
Works great but I cannot say easy to do, ETHOS is about easy :)

**Comment by hrenz** [comment]
like this as a table
Switch Weigth Expo
SA^ ..... 80% ..... 55%
SA-  ..... 60% ..... 45%
SAv .... .50% ..... 30%    
and so on

**Comment by hrenz** [comment]
instead of expo you can select a curve so you have (weight), expo and diff in one 
if you like it

**Comment by EngelMT** [comment]
@hrenz Right, this will be possible. But I would be in favour of leaving a clear line in the software. So separate weighting, separate EXPO curves and separate differentiation.

**Comment by centertrim** [comment]
I agree that Expo AND Differential should be added to the switchable values alongside weight.  

Developers:  Please also make the order of operations that you use for the mixer output calculations clear in the manual.  Example might be:  Mixer Input * Weight -> Expo -> Curve -> Diff -> Mixer Output  ??? I am not sure what order of operations you used for all the possible operations, but whatever it is should be clearly stated somewhere.
